lib/logflare/sources/source/bigquery/pipeline.ex

```diff
@@ -147,7 +149,20 @@ defmodule Logflare.Sources.Source.BigQuery.Pipeline do
     OpenTelemetry.Tracer.with_span :bigquery_pipeline, %{
       attributes: Map.new(attributes)
     } do
-      stream_batch(context, messages)
+      if context.bq_storage_write_api do
```

Comment:
If user changes this at runtime then the value would become stale. Might be better to pull source value from cache.